examples/imgui_impl_dx12.cpp

```diff
@@ -552,8 +553,8 @@ bool    ImGui_ImplDX12_CreateDeviceObjects()
         desc.RenderTarget[0].SrcBlend = D3D12_BLEND_SRC_ALPHA;
         desc.RenderTarget[0].DestBlend = D3D12_BLEND_INV_SRC_ALPHA;
         desc.RenderTarget[0].BlendOp = D3D12_BLEND_OP_ADD;
-        desc.RenderTarget[0].SrcBlendAlpha = D3D12_BLEND_INV_SRC_ALPHA;
-        desc.RenderTarget[0].DestBlendAlpha = D3D12_BLEND_ZERO;
+        desc.RenderTarget[0].SrcBlendAlpha = D3D12_BLEND_ONE;
```

Comment:
I intended to write that `D3D12_BLEND_SRC_ALPHA` would be consistent with other backends, but...

It appears backends has different idea how to blend alpha channel into final image:

| Backend | Alpha Channel Blend Equation             |
|---------|------------------------------------------|
| D3D9    | `src * SRC_ALPHA + dest * INV_SRC_ALPHA` |
| D3D10   | `src * INV_SRC_ALPHA + dest * 0`         |
| D3D11   | `src * INV_SRC_ALPHA + dest * 0`         |
| D3D12   | `src * 1 + dest * INV_SRC_ALPHA` |
| OpenGL  | `src * SRC_ALPHA + dest * INV_SRC_ALPHA` |
| Metal   | `src * SRC_ALPHA + dest * INV_SRC_ALPHA` |
| Vulkan  | `src * INV_SRC_ALPHA + dest * 0`         |
| Allegro | `src * SRC_ALPHA + dest * INV_SRC_ALPHA` |

I guess it does not matter until ImGui is rendered into the texture.
What is desired equation?

Reply:
result alpha = 1 - result transparency
result transparency = src transparency * dest transparency
src transparency = 1 - src alpha
dest transparency = 1 - dst alpha

so
result alpha = src alpha * 1 + dest alpha * (1 - src alpha)

D3D12 is correct